sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -76,50 +76,69 @@ export interface ChatClientOptions extends PipelineOptions {
 }
 
 // @public
-export interface ChatMessage extends Omit<RestChatMessage, "senderId" | "content"> {
+export interface ChatMessage {
     content?: ChatMessageContent;
-    sender?: CommunicationUserIdentifier;
+    createdOn: Date;
+    deletedOn?: Date;
+    editedOn?: Date;
+    id: string;
+    sender?: CommunicationIdentifier;
+    senderDisplayName?: string;
+    sequenceId: string;
+    type: ChatMessageType;
+    version: string;
 }
 
-// @public (undocumented)
-export interface ChatMessageContent extends Omit<RestChatMessageContent, "participants"> {
+// @public
+export interface ChatMessageContent {
+    initiator?: CommunicationIdentifier;
+    message?: string;
     participants?: ChatParticipant[];
+    topic?: string;
 }
 
 // @public
-export interface ChatMessageReadReceipt extends Omit<RestChatMessageReadReceipt, "senderId"> {
-    readonly sender: CommunicationUserIdentifier;
+export interface ChatMessageReadReceipt {
+    chatMessageId: string;
+    readOn: Date;
+    sender: CommunicationIdentifier;
 }
 
 // @public
 export type ChatMessageType = string;
 
 // @public
-export interface ChatParticipant extends Omit<RestChatParticipant, "id"> {
-    user: CommunicationUserIdentifier;
+export interface ChatParticipant {
+    displayName?: string;
+    id: CommunicationIdentifier;
```

Comment:
```suggestion
    id: CommunicationIdentifierKind;
```

Reply:
The ChatParticipant is also used as input. Do we want different ChatThread, `ChatParticipant` model? seems very confusing to me in that case